.ksql/03_POSTAL_SNAPSHOT_OSLO_STREAM_FLATTEN_GRB.ksql

```diff
@@ -6,13 +6,13 @@ CREATE OR REPLACE STREAM IF NOT EXISTS postal_snapshot_oslo_stream_flatten_grb
       IDENTIFICATOR->ID Id,
       IDENTIFICATOR->VERSIEID Versieid,
       IDENTIFICATOR->OBJECTID Postcode,
-      IFNULL(GEMEENTE->OBJECTID, '') Niscode,
+      IFNULL(GEMEENTE->OBJECTID, '') Niscode, Postinfo,
```

Comment:
`NisCode, Postinfo` probably postinfo not needed here

Reply:
Fixed